### PR TITLE
Feature/323-cleanup

### PR DESCRIPTION
**Pull request by umulmrum**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | chameleon-system/chameleon-system#323
| License       | MIT

The last batch of removed deprecated code. I also added a second commit that cleans some unused code.

### CODE REVIEW

File: src/CoreBundle/private/library/classes/TCMSListManager/TCMSListManagerMediaSelector.class.php [hunk 1]
```diff
@@ -205,12 +205,6 @@ protected function GetMaxHeight()
 
         if ($this->oGlobal->UserDataExists('imageHeight')) {
             $iHeight = $this->oGlobal->GetUserData('imageHeight');
-        } else {
-            if (isset($this->oDefaultImage->aData) && is_array($this->oDefaultImage->aData)) {
-                if (array_key_exists('height', $this->oDefaultImage->aData) && $this->oDefaultImage->aData['height'] > 0) {
-                    $iWidth = $this->oDefaultImage->aData['height'];
```
Comment (UlrichKu) [inline src/CoreBundle/private/library/classes/TCMSListManager/TCMSListManagerMediaSelector.class.php]: Why is this removed?

The wrong variable name ($iWidth instead of $iHeight) shows that this was not used?

Comment (umulmrum) [reply src/CoreBundle/private/library/classes/TCMSListManager/TCMSListManagerMediaSelector.class.php]: Ah right, I didn't see that the wrong name led to the unused-message. This has been unchanged for almost 9 years now, so I'm inclined to just keep it removed, as no one seems to complain.

Comment (UlrichKu) [reply src/CoreBundle/private/library/classes/TCMSListManager/TCMSListManagerMediaSelector.class.php]: The documentation should be adapted then.

Comment (umulmrum) [reply src/CoreBundle/private/library/classes/TCMSListManager/TCMSListManagerMediaSelector.class.php]: Done.
